tools/depends/native/TexturePacker/src/decoder/GifHelper.h

```diff
@@ -101,15 +101,15 @@ class GifHelper
 
 private:
   std::vector<FramePtr> m_frames;
-  unsigned int    m_imageSize;
-  unsigned int    m_pitch;
-  unsigned int    m_loops;
-  unsigned int    m_numFrames;
+  unsigned int m_imageSize = 0;
+  unsigned int m_pitch = 0;
+  unsigned int m_loops = 0;
+  unsigned int m_numFrames = 0;
 
   std::string     m_filename;
-  GifFileType*    m_gif;
+  GifFileType* m_gif = nullptr;
   std::vector<GifColor> m_globalPalette;
-  unsigned char*  m_pTemplate;
+  unsigned char* m_pTemplate = nullptr;
```

Comment:
Shouldn't we also init `m_gifFile` with a nullptr here (line 113)?

Reply:
Have a look at the `GifHelper` constructor.
https://github.com/xbmc/xbmc/blob/dd178e70a87226656ecbc286ff016f419ad4fce9/tools/depends/native/TexturePacker/src/decoder/GifHelper.cpp#L46-L56